gossip/gossip.go

```diff
@@ -103,6 +103,13 @@ var (
 
 func init() {
 	gob.Register(&proto.NodeDescriptor{})
+	gob.Register(&proto.ZoneConfig{})
```

Comment:
Isn't it the case that they're only useful as long as gossip uses the gob encoder/decoder?